src/main/resources/assets/securitycraft/lang/es_es.json

```diff
@@ -685,8 +685,8 @@
 	"config.securitycraft:ableToCraftKeycard5.tooltip": "¿Se puede fabricar la tarjeta de acceso de nivel 5?",
 	"config.securitycraft:ableToCraftLUKeycard": "¿Tarjeta de acceso de usos limitados fabricable?",
 	"config.securitycraft:ableToCraftLUKeycard.tooltip": "¿Se puede fabricar la tarjeta de acceso de usos limitados?",
-	"config.securitycraft:smallerMineExplosion": "¿Las minas explotan menos?",
-	"config.securitycraft:smallerMineExplosion.tooltip": "Las minas deberían explotar menos fuerte de lo habitual.",
+	"config.securitycraft:smallerMineExplosion": "¿Las minas provocan una explosión más pequeña?",
+	"config.securitycraft:smallerMineExplosion.tooltip": "Las minas deberían provocar una explosión más pequeña de lo habitual.",
```

Comment:
Taking a second look at this and just to keep the style among all config tooltips, please add interrogation marks here too